embulk-output-jdbc/src/main/java/org/embulk/output/jdbc/AbstractJdbcOutputPlugin.java

```diff
@@ -760,6 +761,22 @@ private void dropTables() throws SQLException
         }
     }
 
+    @VisibleForTesting
+    static int calculateSuffixLength(int taskCount)
+    {
+        assert(taskCount >= 0);
+        return Math.max(3, String.valueOf(taskCount - 1).length());
```

Comment:
@civitaspo 
It will be more readable if there is a comment about the reason for the magic number `3` (for backward compatilibity).

A commreason

Reply:
Oops, I'm sorry. You're right. I added a commit https://github.com/embulk/embulk-output-jdbc/pull/301/commits/89a63aabee57e4bc105538a639e68c3243465fc0. Please review again.